### ISSUE TITLE
Changes arch from Linux/X86_64

### ISSUE THREAD

**Issue by danielstravito** [CircleCI-Public/aws-ecs-orb#147]
## Orb version:

2.2.1

## What happened:

I had a taskdef with the following, and the service was live and healthy with ARM64.

```yaml
    "runtimePlatform": {
        "cpuArchitecture": "ARM64",
        "operatingSystemFamily": "LINUX"
    },
```

I then deployed via this orb, and now it's on `Linux/X86_64` rather than `ARM64`.

I use the following for deploys:
```yaml
- aws-ecs/update-service:
          verify-revision-is-deployed: true
          container-image-name-updates: "container=blah,tag=${CIRCLE_SHA1}"
          family: "blah-task"
          cluster-name: "blah-cluster"
          service-name: "blah-service"
```

## Expected behavior:

Either don't touch the platform, or expose functionality that lets you control the platform.

**Comment by danielstravito** [comment]
This script needs to be changed:

https://github.com/CircleCI-Public/aws-ecs-orb/blob/master/src/scripts/register-new-task-def.sh

Needs a parameter for adding `--runtime-platform=cpuArchitecture=ARM64`

**Comment by brivu** [comment]
Hey @danielstravito, 

I've merged PR #155 to add the `runtime-platform` parameter. I'm closing this issue for now but please feel free to reopen if that doesn't work for your use case. 

Thanks! 
Brian

**Comment by danielstravito** [comment]
Shouldn't this be added to the update-service? Or how does it work? Can I now update the taskdef to arm64 and using update-service will no longer overwrite that?
